docs/source/restore/index.rst

```diff
@@ -6,162 +6,79 @@ Restore
+* Parallel restore
+* Control over restore granularity
+* Dry run
+* Progress tracking
+* Pause and resume
 
-   # host_id specifies a mapping from the clone cluster node IP to the source
-   # cluster host IDs.
-   host_id:
-     75.237.179.49: 9b6b5206-63c8-4cdf-9b4f-23e37977fd14
-     75.196.92.133: 2d8045e7-51a4-42c4-bb8f-5e8aa77a8228
-   # destination IP: source ID
-
-The ``hosts`` file would look like:
-
-.. code::
-
-   75.237.179.49
-   75.196.92.133
-   
-
-IP to host ID mapping
-.....................
-
-SSH to one of the nodes and execute the following command:
-
-.. code-block:: none
-
-   scylla-manager-agent download-files -L <backup-location> --list-nodes
-
-it gives you information about all the clusters and nodes available in the backup location.
-
-Example output:
-
-.. code-block:: none
-
-   Cluster: prod (a9dcc6e1-17dc-4520-9e03-0a92011c823c)
-   AWS_EU_CENTRAL_1:
-   - 18.198.164.180 (7e68421b-acb1-44a7-a1a8-af7eaf1bb482)
-   - 3.122.35.120 (adc0a3ce-dade-4672-981e-26f91a3d35cb)
-   - 3.127.175.215 (2a575244-3e3c-44a1-a526-da4394f9525e)
-   - 3.65.85.108 (82f0f486-370d-4cfd-90ac-46464c8012cb)
-   - 3.66.209.145 (9ee92c19-5f78-4865-a287-980218963d96)
-   - 3.66.44.239 (aff05f79-7c69-4ecf-a827-5ea790a0fdc6)
-
-   Cluster: test (da5721cd-e2eb-4d10-a3a7-f729b8f72abf)
-   AWS_EU_CENTRAL_1:
-   - 3.123.55.215 (4001206a-3377-40cb-abd4-d38aad5dec41)
-   - 3.127.1.189 (c6466011-02f9-49dd-8951-c32028dfc6f1)
-   - 3.64.219.214 (bc39bb07-7a21-41cd-b576-51d44c1a694a)
-
-For each node IP in the new cluster you need to assign a host ID (UUID in the listing above).
-The mapping must be put into ``host_id`` variable in ``vars.yaml`` file.
-
-Snapshot tag
-............
+Process
+=======
 
-To list available snapshot tags for a node use ``--list-snapshots`` flag.
+Restore procedure works iteratively over restored locations, manifests and tables.
 
-.. code-block:: none
+    * For each backup location:
 
-   scylla-manager-agent download-files -L <backup-location> --list-snapshots -n <host-id>
+      * Await schema agreement
+      * Find live Scylla nodes that will be used for restoring current location
 
-You can filter snapshot tags containing a specific keyspaces or tables by using glob patterns.
-The parameter ``-K, --keyspace <list of glob patterns to find keyspaces>`` lets you do that.
-It works the same way as in scheduling backups or repairs with Scylla Manager.
+        * Find nodes local to location
+        * If non can be found, find any node with location access
 
-The snapshot ID must be put into ``snapshot_tag`` variable in ``vars.yaml`` file.
+      * List backup manifests for specified snapshot tag
+    * For each manifest:
 
-Inventory
-.........
+        * Filter relevant tables from the manifest
+        * For each table:
 
-Put public IP addresses of all nodes to ``hosts`` file.
+          * Temporarily disable auto-compaction of current table
+          * Temporarily disable GC grace seconds of current table
```

Comment:
PLACEHOLDER: it may not be temporarily, but we may require end-user to bring this value back after the repair.